package.json

```diff
@@ -22,7 +22,9 @@
     "@types/react": "16.9.15",
     "@types/react-intl": "3.0.0",
```

Comment:
Please refactor this, things like redux-logger, or typescript shoudl be in devdepedencies. Also all things related to typescript or used just for dev mode should be placed in dev dependecies.